torch/testing/_internal/common_methods_invocations.py

```diff
@@ -6218,6 +6239,14 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                    dtypesIfCUDA=all_types_and(torch.bool, torch.half, torch.bfloat16),
                    sample_inputs_func=sample_inputs_logit,
                    safe_casts_outputs=True),
+    OpInfo('where',
+           op=lambda self, condition, other: torch.where(condition, self, other),
```

Comment:
This is going to need a comment; this is because we don't support the input being anything but the first argument, right?

Reply:
Yes. For this specific case we can pass input as condition but since condition is always bool, so we won't gradcheck on it.

Reply:
Can we use gradcheck_wrapper to handle the gradcheck case? Or are there other tests which would still be an issue?

Reply:
Also affect `test_variant_consistency_eager`
https://github.com/pytorch/pytorch/blob/28840b9a447e820bf2e66f6b6cdc880c12d077b7/test/test_ops.py#L303-L307

Reply:
OK